tests/models/sam/test_modeling_sam.py

```diff
@@ -450,6 +451,51 @@ def test_model_from_pretrained(self):
         model = SamModel.from_pretrained(model_name)
         self.assertIsNotNone(model)
 
+    @require_torch_sdpa
+    def test_sdpa_can_compile_dynamic(self):
+        self.skipTest(reason="SAM model can't be compiled dynamic yet")
+
+    @require_torch_sdpa
+    def test_sdpa_can_dispatch_non_composite_models(self):
+        """
```

Comment:
it should be the `composite` model as SAM has several sub configs in the main config. You can add class attribute `_is_composite=True` to the tester class so the appropriate tests are called :)

Reply:
cool, thanks, LGTM! Feel free to ping core maintainer for review :)